services/StorageService.ts

```diff
@@ -68,6 +68,20 @@ export default class StorageService {
     };
   }
 
+  public static getRandomString(): string {
+    const chars = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz_-';
+    const stringLength = 25;
+    // according to nanoID: ~611 trillion years needed, in order to have a 1%
+    //                      probability of at least one collision.
+
+    let result = '';
+    for (let i = 0; i < stringLength; i += 1) {
+      result += chars.charAt(Math.floor(Math.random() * chars.length));
+    }
+
+    return result;
+  }
+
```

Comment:
Why do we need this whole implementation? Can we just return a random uuid instead? Does it need to be a length of 25?

Reply:
I tried to find a random UUID implementation natively. I think nowadays [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) is the standard but it is only supported for node version 19+